recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py35]
+    skip: True  # [py3k]
```

Comment:
No need to change the `skip` here. It would be dropped anyways as we dropped Python 3.4 in `conda-smithy` 2.0.0.